setup.py

```diff
@@ -43,46 +36,36 @@ def get_install_requires():
         'alembic', 'cliff', 'colorlog', 'numpy', 'scipy', 'six',
         'sqlalchemy>=1.1.0', 'tqdm', 'typing',
     ]
-    if sys.version_info[0] == 2:
-        install_requires.extend(['enum34'])
     return install_requires
 
 
 def get_extras_require():
     # type: () -> Dict[str, List[str]]
 
-    _major, _sub = sys.version_info[:2]
-    _fastai_available = (_major == 3) and (_sub > 5)
-
     testing_requirements = [
         'bokeh', 'chainer>=5.0.0', 'cma', 'keras', 'lightgbm', 'mock',
         'mpi4py', 'mxnet', 'pandas', 'plotly>=4.0.0', 'pytest', 'scikit-optimize',
         'tensorflow', 'tensorflow-datasets', 'xgboost', 'scikit-learn>=0.19.0',
-        'torch', 'torchvision', 'pytorch-ignite', 'pytorch-lightning',
+        'torch', 'torchvision', 'pytorch-ignite', 'pytorch-lightning', "fastai<2",
     ]
 
     example_requirements = [
         'chainer', 'keras', 'catboost', 'lightgbm', 'scikit-learn',
         'mxnet', 'xgboost', 'torch', 'torchvision', 'pytorch-ignite',
-        'dask-ml', 'dask[dataframe]', 'pytorch-lightning',
+        'dask-ml', 'dask[dataframe]', 'pytorch-lightning', "fastai<2",
 
         # TODO(Yanase): Update examples to support TensorFlow 2.0.
         # See https://github.com/optuna/optuna/issues/565 for further details.
         'tensorflow<2.0.0',
     ]
-    if _fastai_available:
```

Comment:
It seems that this check is still needed, because fastai doesn't support Python 3.5 but Optuna does.

Reply:
Thanks! You are right.

I fixed it at ea6cfc1.